validator/package.json

```diff
@@ -12,6 +12,7 @@
     "preinstall": "node ../build-system/common/check-package-manager.js"
   },
   "devDependencies": {
+    "google-closure-compiler-java": "20210601.0.0",
```

Comment:
This file originally used `google-closure-compiler`, and not `google-closure-compiler-java`. See old code review thread at https://github.com/ampproject/amphtml/pull/34698/files#diff-66ca7c1acf44670e2e6cf3f1c2980875d6627c39c67e415906cfb520bea72000L15.

I assume the `-java` version is preferable, since it leaves out the other unused native compiler flavors. Calling this out here so we're sure about this choice.

Reply:
If you peek at the validator.py file within that diff, you'll see that it actually referenced the `-java` pacakge.

Reply:
You're right. Leaving this PR as-is sgtm.